src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/model/GPAllocModel.scala

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.dsde.workbench.gpalloc.model
+
+import scala.language.implicitConversions
+
+object BillingProjectStatus extends Enumeration {
```

Comment:
Turning `BillingProjectStatus` into an abstract data type (similarly to `ProjectCreationMonitorMessage` below) might make it easier to follow and extend, if necessary.

Reply:
this is a pretty standard way of doing enums in scala, fwiw